AssetInformationListener/serverless.yml

```
@@ -21,7 +21,7 @@ functions:
     environment:
       TenureApiUrl: ${ssm:/housing-tl/${self:provider.stage}/tenure-api-url}
       TenureApiToken: ${ssm:/housing-tl/${self:provider.stage}/tenure-api-token}
-      AccountApiUrl: ${ssm:/housing-tl/${self:provider.stage}/account-api-url}
+      AccountApiUrl: ${ssm:/housing-finance/${self:provider.stage}/account-api-url}
       AccountApiToken: ${ssm:/housing-finance/${self:provider.stage}/account-api-token}
```

```suggestion
      AccountApiToken: ${ssm:/housing-tl/${self:provider.stage}/account-api-token}
```
I just noticed that [this PR](https://github.com/LBHackney-IT/asset-information-listener/pull/19) updated the wrong path. The API URL apth should be `/housing-finance/` like you have, but the token path should be `/housing-tl/`.

---

I've updated the token path as well :)